tests/test_debugging.py

```diff
@@ -191,7 +191,10 @@ def circuit():
     @pytest.mark.parametrize("diff_method", [None, "parameter-shift"])
     def test_all_state_measurement_snapshot_pure_qubit_dev(self, dev, diff_method):
         """Test that the correct measurement snapshots are returned for different measurement types."""
-        if isinstance(dev, (qml.devices.default_mixed.DefaultMixed, qml.QutritDevice)):
+        if isinstance(
+            dev,
+            (qml.devices.default_mixed.DefaultMixed, qml.QutritDevice),
+        ):
```

Comment:
```suggestion
        if isinstance(dev, (qml.devices.default_mixed.DefaultMixed, qml.QutritDevice)):
```

Reply:
I am using the default settings and `.precommit` didn't complain 🤷